### PR TITLE
Create TheHackerDude1083.json

### PR DESCRIPTION
**Pull request by ShahkerTHD**
### Requirements
- [x] You're not using Vercel or Netlify.
- [x] You have completed your website, there's no type of placeholder at the website. *This requirement is optional if you are registering a domain for emails.*
- [x] The website is reachable.
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.
   - *You should have your email presented. If you don't want to share email, you can leave email an empty string (`""`) and add any other social such as Discord/Twitter/etc.*

### CODE REVIEW

File: domains/TheHackerDude1083.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "description": "Running my new website",
+    "repo": "https://github.com/TheHackerDude1083/TheHackerDude1083.github.io",
+    "owner": {
+        "username": "TheHackerDude1083",
+        "email": "shaheeransari57@gmail.com",
+        "twitter": "<n/a>"
```
Comment (WilliamDavidHarrison) [inline domains/TheHackerDude1083.json:7]:
```suggestion
```

File: domains/TheHackerDude1083.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+    "description": "Running my new website",
+    "repo": "https://github.com/TheHackerDude1083/TheHackerDude1083.github.io",
+    "owner": {
+        "username": "TheHackerDude1083",
+        "email": "shaheeransari57@gmail.com",
```
Comment (WilliamDavidHarrison) [inline domains/TheHackerDude1083.json:6]:
```suggestion
        "email": "shaheeransari57@gmail.com"
```

**Review by wdhdev — CHANGES_REQUESTED** [review]

**Comment by TheHackerDude1083** [comment]
Fixed

**Review by wdhdev — CHANGES_REQUESTED** [review]
404

**Comment by TheHackerDude1083** [comment]
What do I need to fix?

**Comment by WilliamDavidHarrison** [comment]
> What do I need to fix?

Can you provide a link yo your website?

**Comment by TheHackerDude1083** [comment]
https://thehackerdude1083.github.io/My-Website/

**Review by wdhdev — DISMISSED** [review]

**Comment by TheHackerDude1083** [comment]
When I add a custom domain it says "Your site's DNS settings are using a custom subdomain, thehackerdude1083.is-a.dev, that is set up as an A record. We recommend you change this to a CNAME record pointing to thehackerdude1083.github.io. " How do I fix this?

**Comment by WilliamDavidHarrison** [comment]
> When I add a custom domain it says "Your site's DNS settings are using a custom subdomain, thehackerdude1083.is-a.dev, that is set up as an A record. We recommend you change this to a CNAME record pointing to thehackerdude1083.github.io. " How do I fix this?

It says that as your changes haven't been merged yet.  Just wait for this PR to be merged and the message should go away

**Review by andrewstech — APPROVED** [review]

**Pull request merged**